### ISSUE TITLE
args.every is not a function 

### ISSUE THREAD

**Issue by vasanthudhaya** [tungs/timecut#16]
Hi @tungs, When starting my server using js file,  this error occurs. Kindly know me what argument i missed in it. 

```
const timecut = require('timecut');
const fs = require('fs');
timecut({
  headless: false,
  frameCache : true,
  launchArguments : '--no-sandbox --allow-http-screen-capture --disable-setuid-sandbox --single-process',
  url: 'localurl/video',
  timeout: 10000,
  viewport: {
    width: 800,               // sets the viewport (window size) to 800x600
    height: 600
  },
  selector: '#myCanvas',     // crops each frame to the bounding box of '#container'
  left: 20, top: 40,          // further crops the left by 20px, and the top by 40px
  right: 6, bottom: 30,       // and the right by 6px, and the bottom by 30px
  fps: 30,                    // saves 30 frames for each virtual second
  duration: 20,               // for 20 virtual seconds 
  output: 'video.mp4'         // to video.mp4 of the current working directory
}).then(function () {
  console.log('Done!'); 
}).catch(function(e) {
  console.log(e);
})
```

Cc: @varunarora

**Comment by andreastepel** [comment]
Hey @vasanthudhaya! What was your solution? I have the same issue after deploying timecut to Heroku.

**Comment by nguyenhongphat0** [comment]
Hi @andreastepel, I ran into the same problem and turn out `launchArguments` is expecting an array of string, and we're passing a string instead. Fixed it by chaining .split(' ') at the end. Hope it helps :hugs: 

Details: `launchArguments <Array <string>> Extra arguments for Puppeteer/Chromium. Example: ['--single-process']. A list of arguments can be found here.`

**Comment by escully27** [comment]
@andreastepel @nguyenhongphat0 how do you run/install ffmpeg on Heroku? Are you building with Vue? looking to deploy a scalable version of this

**Comment by nguyenhongphat0** [comment]
> how do you run/install ffmpeg on Heroku?

I didn't host my application on Heroku. To install `ffmpeg` I used the system package manager (`apt install ffmpeg`)

> Are you building with Vue?

We use React.JS, but I don't think it's relevant since timeout is running server side 👀

**Comment by escully27** [comment]
Thank you - I would love to run on server too but have only used JS on frontend. Do you have any references or tips on how to do this ? Really appreciate it.

**Comment by nguyenhongphat0** [comment]
You can find really good references and guide on the internet on how to capture smooth browser animation using this library, such as this one: https://medium.com/@stevetung/recording-javascript-animations-with-timecut-18ec99cf943f

About tips, I used `timeout` once in my project. It was a slideshow maker (user upload some photos and chain them to a beautiful slideshow). We used `anime.js` to make a web player, and some point we need to "export them to raw video such as .mp4" so that they can share outside of our application. Timecut did a great job on rendering a smooth and high quality of video, but the drawbacks is the slow capturing speed, and it don't support CSS animation, so be careful about that!

**Comment by escully27** [comment]
I've checked out that tutorial and it's great. I've tested timecut with command line and local javascript with ffmpeg on my machine.

I'm looking to have a prod environment where users can request videos of webpages to be generated and emailed/saved to them. 

Again locally I've achieved this and timecut is incredible, just what I want, but IDK how to have a public environment (heroku or AWS eb?) where users can do it themselves. I could get ffmpeg on a django python backend but obviously that's not compatible with this JS project. If I were to deploy a Vue project with the javascript package, IDK how to have ffmpeg included on the client. Am I missing something or talking crazy?? Confused haha
